src/test/java/features/239 Initial WinAppDriver Implementation.feature

```diff
@@ -3,46 +3,79 @@
-  	Then I verify the Header contains the text "A New Popup Window"
-  	When I switch to the Encode DNA New Tab Page in the previous window
-  	Then I verify the Header contains the text "Window Opened in a New Tab"
\ No newline at end of file
+    Switch to the Previous Tab and Ensure We Have Switched
+    Open a New Window and Ensures it Opens
+    Switch to the New Tab and Ensures We Have Switched
+    Switch to the New Window and Ensures We Have Switched
+    Switch to the Original Window and Ensures We Have Switched
+
+    Given I am on the Encode DNA Home Page
+    When I click the Open New Tab Button
+    Then I verify a new tab opens to the Encode DNA New Tab Page
+      And I verify the Header contains the text "Window Opened in a New Tab"
+    When I switch to the Encode DNA Home Page on the previous tab
+      And I click the Open New Window Button
+    Then I verify a new window opens the Encode DNA PopUp Window
+      And I verify the Header contains the text "A New Popup Window"
+    When I switch to the Encode DNA New Tab Page in the previous window
+    Then I verify the Header contains the text "Window Opened in a New Tab"
+    When I switch to the Encode DNA PopUp Window in the next window
+    Then I verify the Header contains the text "A New Popup Window"
+    When I switch to the Encode DNA New Tab Page in the previous window
+    Then I verify the Header contains the text "Window Opened in a New Tab"
+
+  @239E
+  Scenario: 239E Verify notepad window has changed after entering text using image comparison
+    Given I switch to the Notepad App
+      And I click the Text Editor field
+      And I clear the Text Editor field
+      And I verify the Text Editor field is empty
+      And I take a screenshot of the window
+    When I enter test in the Text Editor field
+    Then I verify the window does not match the original image
+
+  @239F
+  Scenario: 239F Verify notepad window has not changed when no operations have occurred using image comparison
+    Given I switch to the Notepad App
+      And I click the Text Editor field
+      And I clear the Text Editor field
+      And I verify the Text Editor field is empty
+      And I take a screenshot of the window
+    Then I verify the window matches the original image
+
+  @239G
+  Scenario: 239G Verify notepad text editor field element has changed after entering text using image comparison
+    Given I switch to the Notepad App
+      And I click the Text Editor field
+      And I take a screenshot of the Text Editor field
+    When I enter test in the Text Editor field
+    Then I verify the Text Editor field does not match the original image
+
+  @239H
+  Scenario: 239H Verify notepad text editor field element has not changed when no operations have occurred using image comparison
+    Given I switch to the Notepad App
+      And I take a screenshot of the Text Editor Field
+    Then I verify the Text editor Field matches the original image
```

Comment:
This is a lot of testing time for a couple validations. While I do like atomic tests, this level of granularity is not necessary in this case. We are specifically testing Our WinAppDriver tests. Combining these tests will create a real world example, and reduce the total test time significantly cause we don't have to keep re-opening Notepad. It will also test all your inherent assumptions about naming convention and make sure you don''t have any hidden name collision issues.

Reply:
We'll need to change how we handle screenshots. As of right now if these were combined we'd overwrite the previous screenshot's results each time a comparison occured.